stdlib/format.ml

```diff
@@ -794,7 +701,7 @@ let pp_close_tbox state () =
   if state.pp_curr_depth > 1 then
   begin
    if state.pp_curr_depth < state.pp_max_boxes then
-     let elem = make_queue_elem (size_of_int 0) Pp_tend 0 in
+     let elem = { size = Size.zero; token = Pp_tend; length = 0 } in
```

Comment:
We had many `{ size = Size.zero; token; length = 0 }` repetitions already, maybe you could use an auxiliary function?

```ocaml
let zero_token token = { size = Size.zero; token. length = 0 }
```

Reply:
I tried `zero_token`, but settled on re-introducing `make_queue_elem` but with optional size and length defaulting to zero. I rewrote the history a bit to hide the back-and-forth.

If I ever come up with a good name for "pretty printing queue element" (which is different from token), I will probably introduce a module for it with `make_queue_elem` becoming `create`.

Reply:
Actually I'm not fond of this new approach, because it makes it easy for programmers to write `make_queue_elem` and *not think* about the size and length they want for their token. Explicit record initialize forced them to think, and `zero_token` has a name that clearly indicates that it should only be used in some special situation.

If you had a systematic way to compute the size and length from the token (it looks like it might be the case from the code), then you could make those arguments go away from `make_queue_elem` in a way that is safe, because it always make the right choice. But hiding a choice that can be wrong is not good UI.